<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PinUnblock</class>
 <widget class="QDialog" name="PinUnblock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>381</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>557</width>
    <height>381</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 1px solid #5e5e5e;
background-color: #e8e8e8;</string>
  </property>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>336</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 6px 10px;
border-radius: 3px;
background-color: #c53e3e;
font-family: Open Sans;
font-size: 14px;
color: #ffffff;
font-weight: 400;
text-align: center;</string>
   </property>
   <property name="text">
    <string>Katkesta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="unblock">
   <property name="geometry">
    <rect>
     <x>299</x>
     <y>336</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Blokeeri lahti</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameId">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>16</y>
     <width>208</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
font-family: Arial;
font-size: 16px;
color: #000000;
font-weight: 700;
line-height: 17px;
text-align: center;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>46</y>
     <width>528</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
font-family: Arial;
font-size: 14px;
color: #000000;
font-weight: 400;
line-height: 16px;
</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetLabelPuk" native="true">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>138</y>
     <width>300</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="labelPuk">
      <property name="styleSheet">
       <string notr="true">font-family: Arial;
font-size: 14px;
color: #000000;
font-weight: 400;
line-height: 16px;</string>
      </property>
      <property name="text">
       <string>Kehtiv PUK kood</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="iconLabelPuk" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="spacerLabelPuk">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="puk">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>156</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 0px 10px;
border: 1px solid #c8c8c8;
background-color: #ffffff;
font-family: Arial;
font-size: 14px;
color: #000000;
font-weight: 400;
line-height: 14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widgetLabelPin" native="true">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>200</y>
     <width>300</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="labelPin">
      <property name="styleSheet">
       <string notr="true">font-family: Arial;
font-size: 14px;
color: #000000;
font-weight: 400;
line-height: 16px;</string>
      </property>
      <property name="text">
       <string>pin</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="iconLabelPin" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="spacerLabelPin">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="pin">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>218</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 0px 10px;
border: 1px solid #c8c8c8;
background-color: #ffffff;
font-family: Arial;
font-size: 14px;
color: #000000;
font-weight: 400;
line-height: 14px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QWidget" name="widgetLabelRepeat" native="true">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>261</y>
     <width>300</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="labelRepeat">
      <property name="styleSheet">
       <string notr="true">font-family: Arial;
font-size: 14px;
color: #000000;
font-weight: 400;
line-height: 16px;</string>
      </property>
      <property name="text">
       <string>repeat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="iconLabelRepeat" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="spacerLabelRepeat">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="repeat">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>279</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 0px 10px;
border: 1px solid #c8c8c8;
background-color: #ffffff;
font-family: Arial;
font-size: 14px;
color: #000000;
font-weight: 400;
line-height: 14px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
